classes/submission/reviewAssignment/maps/Schema.php

```diff
@@ -52,9 +55,21 @@ public function summarize(ReviewAssignment $item): array
     public function mapMany(Enumerable $collection): Enumerable
     {
         $this->collection = $collection;
-        return $collection->map(function ($item) {
-            return $this->map($item);
-        });
+        $this->submissions = Repo::submission()->getCollector()
+            ->filterByContextIds([$this->context->getId()])
```

Comment:
I didn't inspect how this code is used, but the `$this->context` is nullable, so it's better to add some protection either here or on the constructor (i.e. force it to not be null).

Reply:
Hmm, I see this pattern is used in other schemas, e.g., submission. I'm going to leave this as is - I'm not sure if we should allow context wild card here.

Reply:
I've just suggested to add defensive code, because the parent class (`PKP\core\maps\Schema`) allows a null context to be passed, otherwise an exception might happen.

```php
->filterByContextIds($this->context ? [$this->context->getId()] : [])
```

p.s.: There's a protection to not load data for all contexts, if that's what you mean with "context wildcard". You've got to pass `Application::CONTEXT_ID_ALL` which translates to `_` (🤨).

Reply:
I'll open another issue

Reply:
https://github.com/pkp/pkp-lib/issues/9944